app/packages/app/src/pages/datasets/DatasetPage.tsx

```diff
@@ -15,6 +15,7 @@ import Nav from "../../components/Nav";
 import { Route } from "../../routing";
 import style from "../index.module.css";
 import { DatasetPageQuery } from "./__generated__/DatasetPageQuery.graphql";
+import Starter from "@fiftyone/core/src/components/Starter";
```

Comment:
Small import nit. It's good to be explicit about what packages the core library exports I think
```suggestion
import { Starter } from "@fiftyone/core";
```